fact-ebpf/src/bpf/maps.h

```diff
@@ -42,7 +42,7 @@ struct {
  */
 struct path_cfg_helper_t {
   unsigned int bit_len;
-  char path[PATH_MAX];
+  char path[PATH_MAX * 2];
```

Comment:
Based on our slack discussion about this, it'd be good to have a comment here about why the increase of size to avoid verifier problems, despite the code never using more than 4096 bytes